### PR TITLE
scrollview: public update() should call recalculate element sizes

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

**Review by KuznetsovVN — DISMISSED** [review]

File: js/renovation/ui/scroll_view/scrollable_native.tsx [hunk 1]
```diff
@@ -894,7 +892,9 @@ export class ScrollableNative extends JSXComponent<ScrollableNativePropsType>()
       return false;
     }
 
-    this.update();
+    if (!this.props.updateManually) {
```
Comment (IgnatovDan) [inline js/renovation/ui/scroll_view/scrollable_native.tsx]: Condition seems to be incorrect:
- the 'validate' method should do validation without additional actions, additional actions should be out of this method
- tryGetAllowedDirection/isScrollingOutOfBound are called after 'update' is completed and their result can be changed if 'update' was not called.

Comment (IgnatovDan) [reply js/renovation/ui/scroll_view/scrollable_native.tsx:895]: Can we remove it?
hg/mobile/rev/c52851107602

Comment (IgnatovDan) [reply js/renovation/ui/scroll_view/scrollable_native.tsx]: Can we remove updateManually?
hg/mobile/rev/c52851107602

**Review by IgnatovDan — DISMISSED** [review]

File: js/renovation/ui/scroll_view/scrollable_native.tsx (hunk 1 above)
Comment (EugeniyKiyashko) [reply js/renovation/ui/scroll_view/scrollable_native.tsx]: fixed in c4f2bdc

**Review by novsstation — DISMISSED** [review]

**Review by KuznetsovVN — DISMISSED** [review]

**Review by novsstation — APPROVED** [review]

**Pull request merged**